macOS/DuckDuckGo/Preferences/View/PreferencesRootView.swift

```diff
@@ -130,47 +141,14 @@ enum Preferences {
             .padding(.horizontal, Const.panePaddingHorizontal)
         }
 
-        private func makeSubscriptionViewModel() -> PreferencesSubscriptionModel {
-            let openURL: (URL) -> Void = { url in
-                DispatchQueue.main.async {
-                    WindowControllersManager.shared.showTab(with: .subscription(url.appendingParameter(name: AttributionParameter.origin, value: SubscriptionFunnelOrigin.appSettings.rawValue)))
-                }
-            }
-
-            let handleUIEvent: (PreferencesSubscriptionModel.UserEvent) -> Void = { event in
+        private func makePurchaseSubscriptionViewModel() -> PreferencesPurchaseSubscriptionModel {
+            let userEventHandler: (PreferencesPurchaseSubscriptionModel.UserEvent) -> Void = { event in
                 DispatchQueue.main.async {
                     switch event {
-                    case .openVPN:
-                        PixelKit.fire(PrivacyProPixel.privacyProVPNSettings)
-                        NotificationCenter.default.post(name: .ToggleNetworkProtectionInMainWindow, object: self, userInfo: nil)
-                    case .openFeedback:
-                        NotificationCenter.default.post(name: .OpenUnifiedFeedbackForm,
-                                                        object: self,
-                                                        userInfo: UnifiedFeedbackSource.userInfo(source: .ppro))
-                    case .openDB:
-                        PixelKit.fire(PrivacyProPixel.privacyProPersonalInformationRemovalSettings)
-                        WindowControllersManager.shared.showTab(with: .dataBrokerProtection)
-                    case .openITR:
-                        PixelKit.fire(PrivacyProPixel.privacyProIdentityRestorationSettings)
-                        let url = subscriptionManager.url(for: .identityTheftRestoration)
-                        WindowControllersManager.shared.showTab(with: .identityTheftRestoration(url))
-                    case .iHaveASubscriptionClick:
+                    case .didClickIHaveASubscription:
                         PixelKit.fire(PrivacyProPixel.privacyProRestorePurchaseClick)
```

Comment:
The general idea behind these changes here is since we now have split a single preferences subscription view into 4 to:
- define separate user event handlers per each view with a strict `UserEvent` that matches implemented functionality of that view
- get rid of separate `openURLHandler` and instead define `openURL` `UserEvent` action to handle that